documentation/book/proc-configuring-mirror-maker.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+[id='configuring-kafka-bridge-{context}']
+= Configuring Kafka Mirror Maker
+
+Use the properties of the `KafkaMirrorMaker` resource to configure your Kafka Mirror Maker deployment.
+
+You can configure access control for producer and consumer using TLS or SASL authentication.
+This procedure shows a configuration that uses TLS encryption and authentication on the consumer side for access control.
+
+For more information on configuring access control for Kafka Mirror Maker, see xref:assembly-access-configuration-kafka-mirror-maker-{context}[Configuring Kafka Mirror Maker access control].
+
+.Prerequisites
+
+* xref:cluster-operator-str[{ProductName} and Kafka is deployed]
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Edit the `spec` properties for the `KafkaMirrorMaker` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker
+metadata:
+  name: my-mirror-maker
+spec:
+  # ...
+  replicas: 3 <1>
+  consumer:
+    bootstrapServers: my-source-cluster-kafka-bootstrap:9092 <2>
+    groupId: "my-group" <3>
+    numStreams: 2 <4>
+    offsetCommitInterval: 120000 <5>
+    tls: <6>
+    trustedCertificates:
+      - secretName: my-cluster-cluster-cert
+        certificate: ca.crt
+    authentication: <7>
+      type: tls
+      certificateAndKey:
+        secretName: my-source-secret
+        certificate: public.crt
+        key: private.key
+    config:
+      max.poll.records: 100
+      receive.buffer.bytes: 32768
+  producer:
+    bootstrapServers: my-target-cluster-kafka-bootstrap:9092
+    abortOnSendFailure: false <8>
+    config: <9>
+      compression.type: gzip
+      batch.size: 8192
+  whitelist: "my-topic|other-topic" <10>
+  resources: <11>
+    requests:
+      cpu: "8"
+      memory: 64Gi
+    limits:
+      cpu: "12"
+      memory: 128Gi
+  logging: <12>
+    type: inline
+    loggers:
+      _logger.name_: "INFO"
+  readinessProbe: <13>
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  livenessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  metrics: <14>
+    lowercaseOutputName: true
+    rules:
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*><>Count"
+        name: "kafka_server_$1_$2_total"
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*,
+        topic=(.+)><>Count"
+        name: "kafka_server_$1_$2_total"
+        labels:
+          topic: "$3"
+  jvmOptions: <15>
+    "-Xmx": "8g"
+    "-Xms": "8g"
+  image: my-org/my-image:latest <16>
```

Comment:
This is kind of very advanced option. Maybe we should leave it out in this example and leave onl the chapter mentioning it in the properties section? I' a bit afraid it might confuse people to think they need to use this.

Reply:
I've changed to: 

ADVANCED OPTION: Container image configuration, which is recommended only in special situations. 

The sentence links to the Image Configuration section too.

If you prefer we remove, we can.